packages/ng/callout/src/lib/callout/callout.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="callout palette-{{palette}}" [class.mod-S]="size === 's'" [class.mod-tiny]="tiny">
```

Comment:
`s` and `tiny` could both be values of a `size` input.

Reply:
@MarcFairbrother 

FYI:
- `tiny` switches the component to `inline-block` and reduces the `gap`;
- `size` changes the component size otherwise.

Reply:
Yeah I think this is more of a design-related decision (because it also depends on figma mappings)

Reply:
Also, it can be tiny AND small :
![image](https://github.com/LuccaSA/lucca-front/assets/25581936/cdedd351-bd04-4293-859e-db48a4dd1106)

Reply:
Ok, `condensed` might have been a more appropriate naming choice but I don't mind if we stick to what already exists in the design system.